beater/burrowbeat.go

```diff
@@ -11,15 +11,15 @@ import (
 	"github.com/elastic/beats/libbeat/beat"
 	"github.com/elastic/beats/libbeat/common"
 	"github.com/elastic/beats/libbeat/logp"
-	"github.com/elastic/beats/libbeat/publisher"
+	pub "github.com/elastic/beats/libbeat/publisher"
```

Comment:
Lets remove this change. If we migrate this to libbeat 6.x there are more changes that will be necessary. For example, ES 6 deprecates the use of types. This PR is about moving this beat to libbeat 5.x.